Makefile

```diff
@@ -37,9 +37,9 @@ build:
 
 .EXPORT_ALL_VARIABLES:
 help:
-	docker run --env-file .env -it --rm -v "$(PWD)/render:${WORKDIR}/render" ${ORG}/${NAME} -h
+	docker run --env-file .env -it --rm -v "$(PWD)/render:${WORKDIR}/render" -v "$(PWD)/templates:${WORKDIR}/templates" ${ORG}/${NAME} -h
```

Comment:
Why does it need to mount this volume to display the help text?

Reply:
This is needed because the help text generator displays the available argument `choices`, which is generated from the available template themes, so it needed to mount the same source as for the execution for the available template names to be listed.

However, per your comment below about the packaging, if we're having a dedicated `make dev` instead, I'd revert this to match `run`.